### PR TITLE
[WIP] Add support for django 1.11

### PR DESCRIPTION
**Pull request by rohithasrk**

### CODE REVIEW

File: django_netjsongraph/tests/test_visualizer.py [hunk 1]
```diff
@@ -31,5 +32,5 @@ def test_detail_uuid_exception(self):
         """
         see https://github.com/interop-dev/django-netjsongraph/issues/4
         """
-        response = self.client.get('/topology/a083b494-8e16-4054-9537-fb9eba914861-wrong/')
+        response = get_topology_or_404('/topology/a083b494-8e16-4054-9537-fb9eba914861-wrong/')
```
Comment (nemesisdesign) [inline django_netjsongraph/tests/test_visualizer.py]: you shall not change this line

File: django_netjsongraph/utils.py [hunk 2]
```diff
@@ -61,5 +62,5 @@ def get_topology_or_404(pk, **kwargs):
     })
     try:
         return get_object_or_404(Topology, **kwargs)
-    except ValueError:
+    except (ValidationError, ValueError) as e:
```
Comment (nemesisdesign) [inline django_netjsongraph/utils.py]: don't add `as e`, is not needed here

File: django_netjsongraph/tests/test_visualizer.py (hunk 1 above)
Comment (rohithasrk) [inline django_netjsongraph/tests/test_visualizer.py]: I see. Does self.client.get automatically call get_topology_or_404?

File: django_netjsongraph/utils.py (hunk 2 above)
Comment (rohithasrk) [inline django_netjsongraph/utils.py]: Yes. Will remove

File: django_netjsongraph/tests/test_visualizer.py (hunk 1 above)
Comment (nemesisdesign) [inline django_netjsongraph/tests/test_visualizer.py]: `self.client.get` simulates an HTTP request to the view, see the [django docs regarding the test client](https://docs.djangoproject.com/en/1.11/topics/testing/tools/)

Comment (rohithasrk) [inline django_netjsongraph/tests/test_visualizer.py]: Oh. I see

**Pull request merged**